internal/commands/vaultsecrets/secrets/create.go

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package secrets
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	preview_secret_service "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/client/secret_service"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/stable/2023-06-13/client/secret_service"
+
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/flagvalue"
+	"github.com/hashicorp/hcp/internal/pkg/format"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+	"github.com/hashicorp/hcp/internal/pkg/iostreams"
+	"github.com/hashicorp/hcp/internal/pkg/profile"
+)
+
+func NewCmdCreate(ctx *cmd.Context, runF func(*CreateOpts) error) *cmd.Command {
+	opts := &CreateOpts{
+		Ctx:           ctx.ShutdownCtx,
+		Profile:       ctx.Profile,
+		IO:            ctx.IO,
+		Output:        ctx.Output,
+		PreviewClient: preview_secret_service.New(ctx.HCP, nil),
+		Client:        secret_service.New(ctx.HCP, nil),
+	}
+
+	cmd := &cmd.Command{
+		Name:      "create",
+		ShortHelp: "Create a new static secret.",
+		LongHelp: heredoc.New(ctx.IO).Must(`
+		The {{ template "mdCodeOrBold" "hcp vault-secrets secrets create" }} command creates a new static secret under an Vault Secrets App.
+
+		Once the secret is created, it can be read using
+		{{ template "mdCodeOrBold" "hcp vault-secrets secrets read" }} subcommand.
+		`),
+		Examples: []cmd.Example{
+			{
+				Preamble: `Create new secret in Vault Secrets application on active profile:`,
+				Command: heredoc.New(ctx.IO, heredoc.WithPreserveNewlines()).Must(`
+				$ hcp vault-secrets secrets create secret_1 --data-file=tmp/secrets1.txt
+				`),
+			},
+			{
+				Preamble: `Create new secret in Vault Secrets application on active profile with pipe:`,
```

Comment:
I think the previous example covers the active profile bit so I would drop that and add more text on the pipe.

```suggestion
				Preamble: `Create a new secret in Vault Secrets application by piping the plaintext secret from a command output:`,
```